vlanBridge.go

```diff
@@ -89,6 +92,11 @@ func (vl *VlanBridge) SwitchDisconnected(sw *ofctrl.OFSwitch) {
 
 // PacketRcvd Handle incoming packet
 func (vl *VlanBridge) PacketRcvd(sw *ofctrl.OFSwitch, pkt *ofctrl.PacketIn) {
+	if pkt.TableId == SRV_PROXY_SNAT_TBL_ID || pkt.TableId == SRV_PROXY_DNAT_TBL_ID {
+		// these are destined to service proxy
+		vl.svcProxy.HandlePkt(pkt)
+	}
```

Comment:
Should we return from here since rest of the handling below doesn't apply to packets destined to service proxy?